style.css

```diff
@@ -107,6 +125,280 @@ body {
+  align-items: center;
+  letter-spacing: 0.03em;
+  background: #ff6b00;
+  color: #fff;
+  padding: 1rem;
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: auto;
+  width: 100%;
+  height: auto 1;
+  padding: 0 24px;
+  background-color: #fff;
+}
+
+.grid .card {
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(38, 38, 38, 0.75%),
+      rgba(38, 38, 38, 0.9) 61.94%
+    ),
+    url(./images/placeholder-1.png);
+  background-repeat: no-repeat;
+  width: 327px;
+  height: 385px;
+  display: grid;
+  grid-template-columns: auto;
+  margin-top: 2rem;
+}
+
+.grid .card .card-text {
+  align-items: center;
+  justify-content: center;
+  padding: 4rem 0 0 2rem;
+  color: #fff;
+}
+
+.about .about-desc h2 {
+  color: #ff6b00;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+}
+
+.grid .card .card-text h2 {
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.about .about-desc p {
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.about .about-desc #connect-2 .btn-active:hover {
+  background-color: #fff;
+  color: #ff6b00;
+}
+
+.grid .card .card-text p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.layout .layout-row .layout-col .text p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+  justify-content: center;
+  padding-right: 15px;
+}
+
+.grid .card ul {
+  display: flex;
+  gap: 1rem;
+  padding: 10px 0 0 16px;
+}
+
+.grid .card ul li {
+  list-style: none;
+}
+
+.grid .card .connect {
+  padding-top: 3rem;
+}
+
+/* THE ABOUT SECTION */
+.about {
+  background: #1c1a19;
+  padding-left: 1rem;
+  width: 100%;
+  height: auto;
+  background-image: url(./images/illustration-about-me-mobile.png);
+  background-repeat: no-repeat;
+  background-position: 210px 215px;
+  padding-top: git70px;
+}
+
+.about .about-desc #connect-2 {
+  margin: 3rem 0 0 0;
+}
+
+.about .about-desc #connect-2 a {
+  color: #fff;
+  background: #ff6b00;
+  padding: 1rem 1.5rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.about .about-col {
+  padding: 11rem 0 10rem 0;
+}
+
+.about .about-col .outline .wto {
+  padding: 2rem 0 0 3rem;
+  justify-content: center;
+  align-items: center;
+}
+
+.about .about-col .outline .wto h3 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.about .about-col .outline .wto .last h3 img {
+  padding-right: 1rem;
+}
+
+.about .about-col .outline .wto .last {
+  padding: 2rem 1rem 1.8rem 1rem;
+}
```

Comment:
# Optional

- [ ] I recommend you to use `text-align: left` on this list to match the given design.